exec/jdbc/src/main/java/org/apache/drill/jdbc/package-info.java

```diff
@@ -19,7 +19,13 @@
 /**
  * JDBC driver for Drill.
  * <p>
- *   Drill's JDBC driver class is {@link org.apache.drill.jdbc.Driver}.
+ *   Drill's JDBC driver class is
+ *   {@link org.apache.drill.jdbc.Driver org.apache.drill.jdbc.Driver}.
```

Comment:
Is this link tag written like this because you want the fully qualified name to be shown as the link text?